aspnetcore/tutorials/signalr.md

```diff
@@ -68,6 +68,7 @@ At the end, you'll have a working chat app:
 
    ```dotnetcli
    dotnet new webapp -o SignalRChat
+   cd SignalRChat
    code -r SignalRChat
```

Comment:
Do this, too ...

```suggestion
code -r
```

Reply:
Adding to previous comment. I am running `code -r` and `code -r .` inside Terminal in macOS